Test_Routes Version Handler assertion fails

this assertion fails when I try to build the project:

https://github.com/jmorganca/ollama/blame/1ca484f67e6f607114496211004942013e5595eb/server/routes_test.go#L74

Error:
```
[GIN] 2023/12/19 - 19:11:41 | 200 |      56.659µs |       127.0.0.1 | GET      "/api/version"
```
[...]
```
--- FAIL: Test_Routes (0.00s)
    routes_test.go:185: Running Test: [Version Handler]
    routes_test.go:74: 
                Error Trace:    /build/source/server/routes_test.go:74
                                                        /build/source/server/routes_test.go:199
                Error:          Not equal: 
                                expected: "{\"version\":\"0.0.0\"}"
                                actual  : "{\"version\":\"0.1.17\"}"
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -{"version":"0.0.0"}
                                +{"version":"0.1.17"}
                Test:           Test_Routes
```

Either I don't fully understand how the assertion is supposed to work and I am doing something wrong or the version string `"0.0.0"` in this assertion should be `"0.1.17"`.

--- a/server/routes_test.go
+++ b/server/routes_test.go
@@ -16,6 +16,7 @@ import (
 
 	"github.com/jmorganca/ollama/api"
 	"github.com/jmorganca/ollama/parser"
+	"github.com/jmorganca/ollama/version"
 )
 
 func setupServer(t *testing.T) (*Server, error) {
@@ -71,7 +72,7 @@ func Test_Routes(t *testing.T) {
 				assert.Equal(t, contentType, "application/json; charset=utf-8")
 				body, err := io.ReadAll(resp.Body)
 				assert.Nil(t, err)
-				assert.Equal(t, `{"version":"0.0.0"}`, string(body))
+				assert.Equal(t, fmt.Sprintf(`{"version":"%s"}`, version.Version), string(body))
 			},
 		},
 		{

